packages/hoppscotch-backend/src/plugins/GQLComplexityPlugin.ts

```diff
@@ -23,12 +24,17 @@ export class GQLComplexityPlugin implements ApolloServerPlugin {
 
     return {
       async didResolveOperation({ request, document }) {
+        const isIntrospectionQuery =
+          request.operationName === 'IntrospectionQuery';
```

Comment:
As observed, `request.operationName` was coming up as `undefined`. Hence, this approach won't fit in.